src/components/WorkerCompensation.js

```diff
@@ -233,37 +233,40 @@ const WorkerCompensation = () => {
-                            while also attracting those aligned with the mission
-                            of DXdao.
-                        </NormalText>
-                    </ItemWrapper>
-                    <ItemWrapper>
-                        <Bullet />
-                        <NormalText>
-                            DXdao hopes to attract the best and most ambitious
-                            minds across the world.
-                        </NormalText>
-                    </ItemWrapper>
-                    <ItemWrapper>
-                        <Bullet />
-                        <NormalText>
-                            Transparency in structure and a clear path for
-                            future growth are important.
-                        </NormalText>
-                    </ItemWrapper>
-                    <ItemWrapper>
-                        <Bullet />
-                        <NormalText>
-                            DXdao workers should be incentivized to drive
-                            financial value to DXD; compensation in DXD is meant
-                            to align worker interests with DXD holders.
-                        </NormalText>
-                    </ItemWrapper>
+                    <SubHeader>
+                        Guiding Tenets
+                        <ItemWrapper marginTop={'1'}>
+                            <Bullet />
+                            <NormalText>
+                                DXdao should offer fair & competitive
+                                compensation, while also attracting those
+                                aligned with the mission of DXdao.
+                            </NormalText>
+                        </ItemWrapper>
+                        <ItemWrapper>
+                            <Bullet />
+                            <NormalText>
+                                DXdao hopes to attract the best and most
+                                ambitious minds across the world.
+                            </NormalText>
+                        </ItemWrapper>
+                        <ItemWrapper>
+                            <Bullet />
+                            <NormalText>
+                                Transparency in structure and a clear path for
+                                future growth are important.
+                            </NormalText>
+                        </ItemWrapper>
+                        <ItemWrapper>
+                            <Bullet />
+                            <NormalText>
+                                DXdao workers should be incentivized to drive
+                                financial value to DXD; compensation in DXD is
+                                meant to align worker interests with DXD
+                                holders.
+                            </NormalText>
+                        </ItemWrapper>
+                    </SubHeader>
```

Comment:
Why is everything wrapper in `SubHeader`?

Reply:
nice catch! That shouldn't be as so